### DISCUSSION TITLE
Can we reject BirdNet's ID & ask it to try again?

### DISCUSSION THREAD

**Discussion by ThickKnee** [mcguirepr89/BirdNET-Pi#539] [Ideas]
Sometimes BirdNet's identifications are suspect or even just plain wrong.

I think there needs to be an opportunity to "tell" BirdNet that it's ID is rejected and ask it to have another go. Perhaps its second ID will be correct.

**Comment by blogdemoi** [comment]
I agree, I just had a Turdus merula identified as a Coturnix coturnix :-)

**Comment by ehpersonal38** [comment]
In my experience the model is pretty deterministic, so any given mp3 will always have the same predictions regardless of how many times it's analyzed. However, the new model is just on the horizon which may have some ability to do this? idk yet

**Comment by mcguirepr89** [comment]
https://github.com/mcguirepr89/BirdNET-Pi/discussions?discussions_q=false

Hello, @ThickKnee and @blogdemoi  -- I'm afraid this topic has been discussed far too much and has been under development for a while, with lots of different approaches and suggestions. While I like the idea of a "do-over" for classifications, I won't be implementing this as an automated feature. I would certainly be happy to merge any development around this as a feature were anyone to accomplish creating this type of interface. Also, if there are any other ideas about how to approach this problem, that would be welcomed at this point, though I would hope you would share those on the older discussions for the topic so that the content is not spread throughout the repo.
